scripts/trustpilot.js

```diff
@@ -0,0 +1,6 @@
+const { exec } = require('child_process')
+const { fetchTrustpilotData } = require('./data/trustpilot-generator');
+
+fetchTrustpilotData(() => {
+    exec(`git add src/data/trustpilot.ts`)
```

Comment:
I don't like exec commands like this... it's "magic" stuff that happens (e.g. whats in source control changes on build) and potentially open to misuse.